src/libs/actions/Report.js

```diff
@@ -1084,6 +1086,36 @@ function deleteReportComment(reportID, reportAction) {
                 };
 
                 Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, reportActionsToMerge);
+            } else {
+                // Update last message information in the report object
+                Onyx.connect({
+                    key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
+                    callback: _.once((reportActionList) => {
```

Comment:
Sorry but the use of `_.once` is very bad here. You are creating n number of subscriptions without releasing them. Where n === no of deleted comments.

Reply:
I'm sorry but `_.once` is required there. If you remove `_.once` and put `console.log` inside that callback, you'll see that the callback is called 2-3 times per message delete call. Can you please explain what exactly is the big downside to this approach? Is it using extra memory?

Reply:
So you can call Onyx.disconnect here after the operation is done. Otherwise, this subscription will remain, and On the next delete operation, both old ones and the new ones will be called. And I believe that this _.once will not be required then. But it is possible that while you are listening to this key, other factors update the Store, and thus this callback fire multiple times.
So yeah just `Onyx.disconnect` will be fine. `_.once` has not drawbacks just I was thinking that it won't be required.

Reply:
Thanks. I'll check that out and update PR accordingly if that helps.

Reply:
That actually works flawlessly. Thanks.

PR is updated.